src/smart-components/group/add-group/set-users.js

```diff
@@ -0,0 +1,82 @@
+import React, { Fragment, useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Stack,
+  StackItem,
+  Text,
+  TextContent,
+  TextVariants,
+  Title
+} from '@patternfly/react-core';
+import CreatableSelect from 'react-select/creatable';
```

Comment:
Just FYI we are finally adding re-styled react-select to data driven forms with the create able variant. Once its merged i will let you know.

Reply:
@Hyperkid123 - great, thank you!